.github/workflows/deploy.yml

```diff
@@ -4,6 +4,7 @@ on:
   push:
     branches:
       - main
+      - tsmith/psycopg_fix
```

Comment:
is this just for testing?

Reply:
just for testing. I'll remove it before submitting